### PR TITLE
BT-700 Landing page readme

### PR DESCRIPTION
**Pull request by rick-bt**
BT-700 Landing page text for the backtrace-javascript repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+
+<div>
+    <div style="float: left; width: 50%;">
+        <ul>
+        <li>Advanced breadcrumbs</li>
+        <li>Attachments</li>
+        <li>Framework-specific attributes</li>
+        <li>Stability metrics</li>
+        </ul>
+    </div>
```
Comment (konraddysput) [inline README.md:19]: why break here?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+<p align="center">
+    <a href="https://backtrace.io/" target="_blank">
+        <img src="./sl.bt.png" alt="Backtrace" width="512">
+    </a>
+</p>
 
-Home of the all Backtrace's JavaScript integrations.
+# Sauce Labs backtrace-javscript SDKs
+
+These Javascript SDKs are a radical step forward in error reporting for web applications, with a number of exciting new features.
```
Comment (konraddysput) [inline README.md:9]: Let's say it's one year from now. Is this description still valid?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
-# Backtrace-Javascript
+<p align="center">
+    <a href="https://backtrace.io/" target="_blank">
```
Comment (konraddysput) [inline README.md:2]: Backtrace logo color is weird. Are you sure this is a valid Backtrace logo?

**Review by konraddysput — COMMENTED** [review]
Left some comments. Lets address them before making a decision if we should land/request changes

File: README.md (hunk 1 above)
Comment (rick-bt) [reply README.md:19]: The extra div clears the float.

File: README.md (hunk 3 above)
Comment (rick-bt) [reply README.md:2]: We cannot set a different image for light vs. dark mode. I constructed this one to handle both. Too weird? Not weird enough?

File: README.md (hunk 2 above)
Comment (rick-bt) [reply README.md:9]: Nope. And overall i don't like it. Working on new copy.

File: README.md (hunk 3 above)
Comment (rick-bt) [reply README.md]: Implemented a deprecated method to support light/dark images in GitHub. I expect it will continue to work fine.

**Pull request merged**